packages/components/src/components/DetailsSearch.vue

```diff
@@ -179,6 +210,13 @@ export default {
     selectObject(type, object) {
       if (type === 'labels') {
         this.$store.commit(SELECT_LABEL, object);
+      } else if (type === 'analysis-finding') {
+        const { events } = object;
+        if (events && events.length >= 1) {
+          this.$store.commit(SELECT_OBJECT, events);
```

Comment:
I'd expect to be brought to finding details, not an event. A finding may consist of many events and the finding details panel should help us navigate them.